pkg/apps/application.jsx

```diff
@@ -80,11 +80,11 @@ class Application extends React.Component {
 
             return description.map(paragraph => {
                 if (paragraph.tag == 'ul') {
-                    return <ul>{paragraph.items.map(item => <li>{item}</li>)}</ul>;
+                    return <ul key={paragraph}>{paragraph.items.map(item => <li key={item}>{item}</li>)}</ul>;
```

Comment:
This would  repeat the entire paragraph text in the HTML, which seems rather useless and wasteful? What is the purpose of all these keys?

Reply:
Discussed in #9339 , the long paragraph text will not make it to the HTML

Reply:
But `paragraph` is not a string...

Reply:
replacing with string `paragraph-${index}`. Using indexes is not preferred, but not anti-pattern and working.